opentelemetry-user-events-metrics/src/tracepoint/mod.rs

```diff
@@ -79,5 +80,36 @@ pub unsafe fn register(trace_point: Pin<&ehi::TracepointState>) -> i32 {
     debug_assert!(METRICS_EVENT_DEF[METRICS_EVENT_DEF.len() - 1] == b'\0');
 
     // CStr::from_bytes_with_nul_unchecked is ok because METRICS_EVENT_DEF ends with "\0".
-    trace_point.register(ffi::CStr::from_bytes_with_nul_unchecked(METRICS_EVENT_DEF))
+    // Returns errno code 95 if trace/debug file systems are not mounted
+    // Returns errno code 13 if insufficient permissions
+    // If tracepoint doesn't exist, it will create one automatically
+    let result = panic::catch_unwind(|| {
+        // CStr::from_bytes_with_nul_unchecked is ok because METRICS_EVENT_DEF ends with "\0".
+        trace_point.register(ffi::CStr::from_bytes_with_nul_unchecked(METRICS_EVENT_DEF))
+    });
+
+    match result {
+        Ok(value) => {
+            if value == 0 {
+                println!("Tracepoint registered successfully.")
```

Comment:
nit: hint/comment that this is done as a temporary change to quickly test (user_events is very new to Kernel itself)), and will be replace/removed with proper internal logging mechanism.